ci-operator/step-registry/telcov10n/system/telco-kpis/run-reboot-test/telcov10n-system-telco-kpis-run-reboot-test-commands.sh

```
@@ -0,0 +1,118 @@
+        exit 1
+    fi
+
+    cd /eco-ci-cd
+
+    echo "Running collect-node-info playbook for spoke ${SPOKE_CLUSTER}"
+    ansible-playbook ./playbooks/telco-kpis/collect-node-info.yml \
+        -i ./inventories/ocp-deployment/build-inventory.py \
+        -e spoke_cluster="${SPOKE_CLUSTER}" \
+        -e spoke_kubeconfig="${SPOKE_KUBECONFIG}" \
+        -e skip_rebuild_image="${SKIP_REBUILD_IMAGE}" \
+        $([ "${DEBUG}" = "true" ] && echo "-vvv")
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)